### ISSUE TITLE
Looking where are saved profiles. 

### ISSUE THREAD

**Issue by zdunex25** [vipersaudio/viper4android_fx#71]
I recently updated from kitkat to marshmallow and I cannot import previously saved profile, import list is empty. Tried to save a new profile to see if any new directory will show up, but it didn't.

Would appreciate a path so it could copy my profile and finally import it.

**Comment by zdunex25** [comment]
/data/media/legacy it is here, but it's old mount point for older API. Not visible by regular file manager.

**Comment by zdunex25** [comment]
This is the problem: https://github.com/vipersaudio/viper4android_fx/blob/master/android_4.x-5.x/src/com/vipercn/viper4android_v2/activity/StaticEnvironment.java#L136

**Comment by timkoers** [comment]
@zdunex25 I completely agree that it should not be coded like this. I vote to reopen this issue. :+1:
